### DISCUSSION TITLE
4/26(수) swift 비동기 처리방식 및 비동기가 어떤 것인지 알아주세요

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#108] [General]
저희는 먹젠 프젝은 무조건 urlsession을 도입할 생각입니다. 이유는 저는 alamofire라는 라이브러리로 시작했기에 URL이 어떻게 돌아가는지 그리고 alamofire 같은 것은 아무래도 라이브러리 보다 보니까 사용방법은 쉽습니다. 네트워크가 필요한 프젝에는 왠만하면 비동기 처리라는 것을 해야하는데 저희는 RX나 컴바인은 사용하지 않을 것이기 때문에 GCD를 사용한 Async, Await? 인가 그거를 쓸 것 같습니다. 솔직히 아직 저도 디스패치큐와 Async, Await?를 다했다고는 장담하지 못하나... 다른 라이브러리로 입문해서 아무것도 못하고 모르는 것보다 낫다고 생각합니다. 시험 끝나고 바로 작업에 들어갈 것이라서 개념은 확실이 알고 있어야한다고 생각합니다. 😁

**Comment by Eunho0922** [comment]
넵
